pkgs/stdenv/generic/setup.sh

```diff
@@ -1337,6 +1337,7 @@ installPhase() {
 
     # shellcheck disable=SC2086
     local flagsArray=(
+        ${enableParallelInstalling:+-j${NIX_BUILD_CORES}}
```

Comment:
Are there other build systems we should be supporting this in?  Ninja, for example?

Reply:
Yeah, extending it to other build systems sounds reasonable.

Reply:
Switched to `draft` for now. Will try to extend it to (or at least audit for relevance):

- pkgs/development/tools/build-managers/bmake/setup-hook.sh
- pkgs/development/tools/build-managers/cmake/setup-hook.sh
- pkgs/development/tools/build-managers/ninja/setup-hook.sh
- pkgs/development/tools/build-managers/scons/setup-hook.sh

Reply:
Attempted to cover:
    - cmake (enabled by default, similar to builds)
    - ninja (set parallelism explicitly, don't rely on default)
    - bmake (enable when requested)
    - scons (enable when requested)
    - meson (set parallelism explicitly, don't rely on default)
    - waf (set parallelism explicitly, don't rely on default)
    - qmake-4/5/6 (enable by default, similar to builds)
    - xorg (always enable, similar to builds)

Reply:
Thanks!